src/features/commands/character/character.ts

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash'
+import { Message } from 'discord.js'
+import { CharacterService } from '../../../services/character/character.service'
+import { MessageManagerService } from '../../../services/message-manager/message-manager.service'
+
+export class CharacterCommand {
```

Comment:
The class name is not matching the file name.

Reply:
Will do!